ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/CreateSymbolStylesFromWebStyles/CreateSymbolStylesFromWebStyles.cpp

```diff
@@ -0,0 +1,166 @@
+// [WriteFile Name=CreateSymbolStylesFromWebStyles, Category=DisplayInformation]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "CreateSymbolStylesFromWebStyles.h"
+
+#include "FeatureLayer.h"
+#include "ServiceFeatureTable.h"
+#include "Map.h"
+#include "MapQuickView.h"
+
+#include "UniqueValueRenderer.h"
+#include "SymbolStyle.h"
+#include "SimpleMarkerSymbol.h"
+
+using namespace Esri::ArcGISRuntime;
+
+CreateSymbolStylesFromWebStyles::CreateSymbolStylesFromWebStyles(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_map(new Map(BasemapStyle::ArcGISNavigation, this))
+{
+  QUrl webStyleLayerUrl = QUrl("http://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/LA_County_Points_of_Interest/FeatureServer/0");
```

Comment:
should always use https

```suggestion
  QUrl webStyleLayerUrl = QUrl("https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/LA_County_Points_of_Interest/FeatureServer/0");
```